src/Repository/Jecoute/LocalSurveyRepository.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace AppBundle\Repository\Jecoute;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\Jecoute\LocalSurvey;
+use AppBundle\Entity\Jecoute\NationalSurvey;
+use AppBundle\Entity\ReferentTag;
+use AppBundle\Repository\ReferentTrait;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\ORM\QueryBuilder;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+class LocalSurveyRepository extends ServiceEntityRepository
+{
+    use ReferentTrait;
+
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, LocalSurvey::class);
+    }
+
+    /**
+     * @return LocalSurvey[]
+     */
+    public function findAllByAdherent(UserInterface $adherent): array
+    {
+        return $this
+            ->createSurveysForAdherentQueryBuilder($adherent)
+            ->getQuery()
+            ->getResult()
+        ;
+    }
+
+    /**
+     * @param Adherent|UserInterface $adherent
+     */
+    public function createSurveysForAdherentQueryBuilder(Adherent $adherent): QueryBuilder
+    {
+        return $this
+            ->createQueryBuilder('survey')
+            ->addSelect('questions')
+            ->innerJoin('survey.questions', 'questions')
+            ->innerJoin('survey.author', 'author')
```

Comment:
Because this query is used to get all surveys of the referent zone of the adherent.
It's not the surveys list of the adherent. The adherent is the user connected on the mobile App.